### ISSUE TITLE
Meeting reminder for in email, displays HTML Tags

### ISSUE THREAD

**Issue by shogunpol** [salesagility/SuiteCRM#2584]
When i create meeting and set up reminders for mailbox, the sent email displays HTML tags in body of message.

#### Issue

This issue is a bug because one of functionality doesnt display message properly.
#### Expected Behavior

The body of email reminder is nice formated, and doesnt display HTML tags.
#### Actual Behavior

When we read email reminder about meeting, HTML tags appear.
#### Steps to Reproduce

1.Create Meeting.
2.Set up email reminder.
3.When email arrived, open it.
4.Observe HTML tags on email body.

#### Context

The HTML tags confuses when read message.

The meetings reminders are one of main functions of the system, it should be high priority.
#### Your Environment

* SuiteCRM Version used:
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Chromium Version 53.0.2785.143 Built on Ubuntu
* Environment name and version (e.g. MySQL, PHP 7): 
* Operating System and version (e.g Ubuntu 16.04): Ubuntu 14.04 (64-bit)

**Comment by pgorod** [comment]
@shogunpol you didn't write your SuiteCRM version, which is it? There were changes in this code in 7.7.6, the PR is #2401, maybe that's where the problem is coming from. 

If you're using a language pack, you need to upgrade it also to match this change.

Please post here the contents of your `include/language/en_us.notify_template.html`, and if possible, a copy of **the source** of the email sent.

**Comment by pgorod** [comment]
Sorry about that, I didn't realize there was another place in the code using those templates...

**Comment by pgorod** [comment]
@samus-aran while you're here, could you please complete the [release notes on the wiki for 7.7.6](https://suitecrm.com/wiki/index.php/Release_notes_7.7.6) and add something like this:

> Fixed and improved HTML handling in notification emails (assignments, invitations, reminders). Language packs must be upgraded to match this change.

It was missing from the release notes and some people were confused with the changes... thanks.

**Comment by joaoinacio** [comment]
The problem still persists apparently, the first email notification is OK however the reminder is shown as plain-text with html tag:
```
<p>
    Title: Meeting - Test  HTML<br />
    When: 17/01/2017 13:45 WET(+00:00)<br />
    Location: <br />
    Created By: Administrator
</p>
```

AFAICT part of the fix at 150e1d2ff336c787d24fa296aef69321a6ccf3a6 that strips tags needs to be applied as well to (to Activities/EmailReminder)
